src/rgw/const.py

```diff
@@ -96,3 +97,14 @@ class RgwEndpoint(Enum):
     MOTR_PROCESS_FID = 'motr my fid'
 
 
+# ConfStore keys of rgw.py
```

Comment:
these constants should be added before any class definition, and close to other related constants.

Reply:
Updated in recent patch